content/vi/docs/reference/glossary/label.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Label
+id: label
+date: 2019-29-11
+full_link: /docs/concepts/overview/working-with-objects/labels
+short_description: >
+  Gán nhãn các đối tượng với những thuộc tính để xác định, những thuộc tính này sẽ có ý nghĩa và liên quan tới người dùng.
```

Comment:
Should be:
Gán nhãn các đối tượng (tags objects) với các thuộc tính xác định, có ý nghĩa và có liên quan đến người dùng.